cinema_service/urls.py

```diff
@@ -4,5 +4,6 @@
 urlpatterns = [
     path("admin/", admin.site.urls),
     path("api/cinema/", include("cinema.urls", namespace="cinema")),
+    path("api/user/", include("user.urls", namespace="user")),
```

Comment:
The `namespace` argument in the `include` function should only be used if the included URLconf has an `app_name` attribute defined. Ensure that `cinema.urls` and `user.urls` have `app_name` set if you intend to use namespaces.